README.md

```diff
@@ -59,10 +60,11 @@ See the file [COPYING](COPYING) for more details.
 
 HISTORY
 -------
+XQF was originally written by Roman Pozlevich from Pereslavl-Zalessky, Russia. Over 17 years of existence the project has had the following major developers:
```

Comment:
I'm not very good in English, but I think something like that is well well better:

> XQF was originally written by Roman Pozlevich in 1998 but maintained up to now and improved for you by a devoted team over the years. XQF is an active project.

@zturtleman: If I remember you are a native english speaker, can you help us to write that? :smile: